koans/AboutApplyingWhatWeHaveLearnt.js

```diff
@@ -32,16 +32,18 @@ describe("About Applying What We Have Learnt", function() {
         }
     }
 
-    expect(productsICanEat.length).toBe(FILL_ME_IN);
+    expect(productsICanEat.length).toBe(1);
   });
 
   it("given I'm allergic to nuts and hate mushrooms, it should find a pizza I can eat (functional)", function () {
 
       var productsICanEat = [];
 
-      /* solve using filter() & all() / any() */
+      productsICanEat.push(products.filter(function(item) {
```

Comment:
This should be `productsICanEat =` not `productsICanEat.push`. While `productsICanEat.length` will be the same, the contents will not be.